### DISCUSSION TITLE
Blurry render

### DISCUSSION THREAD

**Discussion by AC1AN** [nftchef/art-engine#49] [Help + Q&A] [answered]
Hi I'm working on generating 24x24 pixel images, In the first place I was using the hashlips engine ( great tool) but I needed more options like the one for Male/Female characters. It works perfectly in the metadata but the render is very blurry, did I miss something ? output format is 1080/1080 

Thank you

**Comment by nftchef** [comment, marked as answer]
Thank you for letting me know! This is now fixed (in #53 )

You will need to set `smoothing: false` in config.js now, and it should upscale without blurring.

**Reply by AC1AN** [reply]
Hi, thanks for your reply but it didn't work, I had to add first "layerctx.imageSmoothingEnabled = format.smoothing;" in main.js lane 338 to make it works 

Thank you for your time and your work
